sdk/python/featurestore_sample/notebooks/sdk_and_cli/network_isolation/Network Isolation for Feature store.ipynb

```diff
@@ -889,17 +1038,19 @@
    },
    "outputs": [],
    "source": [
-    "!az role assignment create --role \"Storage Blob Data Reader\" --assignee-object-id $uai_principal_id --assignee-principal-type ServicePrincipal --scope $source_data_gen2_container_arm_id"
+    "!az role assignment create --role \"Storage Blob Data Contributor\" --assignee-object-id $uai_principal_id --assignee-principal-type ServicePrincipal --scope $source_data_gen2_container_arm_id"
```

Comment:
role assignment should be done before the copy.

Reply:
This is mentioned in the Step 1b text before code cells. Removed the role assignment in the later part as it is redundant.